.drone-v1.yml

```diff
@@ -144,8 +144,12 @@ steps:
         from_secret: play
       PLAY_HTTP_SECRET_KEY:
         from_secret: PLAY_HTTP_SECRET_KEY
+      S3_AWS_ACCESS_KEY_ID:
+        from_secret: S3_AWS_ACCESS_KEY_ID
+      S3_AWS_SECRET_ACCESS_KEY:
+        from_secret: S3_AWS_SECRET_ACCESS_KEY
     commands:
-      - docker run -m 5g -d --name $${DRONE_COMMIT_SHA} -e PLAY_HTTP_SECRET_KEY=$${PLAY_HTTP_SECRET_KEY} -e ENABLE_STATSD=false -e CONTACT_EMAIL=support@test.com -e OOH_PHONE=012345 -e KEY_CLOAK_URL="https://sso-dev.notprod.homeoffice.gov.uk/auth/admin/realms/drt-notprod" -e JAVA_OPTS="-Xmx4g -Dconfig.resource=application-inmemory.conf" -e USE_API_PAX_NOS=true -e ADJUST_EGATES_USE_BY_U12S=true -e PORT_CODE="test" -e ENV="test" -e FORECAST_MAX_DAYS=2 -e DISPLAY_RED_LIST_INFO=false -e BASE_DOMAIN=drt-test.homeoffice.gov.uk 340268328991.dkr.ecr.eu-west-2.amazonaws.com/drt/drt-v2:B${DRONE_BUILD_NUMBER} $${PLAY}
+      - docker run -m 5g -d --name $${DRONE_COMMIT_SHA} -e S3_AWS_ACCESS_KEY_ID=$${S3_AWS_ACCESS_KEY_ID} -e S3_AWS_SECRET_ACCESS_KEY=$${S3_AWS_SECRET_ACCESS_KEY} -e PLAY_HTTP_SECRET_KEY=$${PLAY_HTTP_SECRET_KEY} -e ENABLE_STATSD=false -e CONTACT_EMAIL=support@test.com -e OOH_PHONE=012345 -e KEY_CLOAK_URL="https://sso-dev.notprod.homeoffice.gov.uk/auth/admin/realms/drt-notprod" -e JAVA_OPTS="-Xmx4g -Dconfig.resource=application-inmemory.conf" -e USE_API_PAX_NOS=true -e ADJUST_EGATES_USE_BY_U12S=true -e PORT_CODE="test" -e ENV="test" -e FORECAST_MAX_DAYS=2 -e DISPLAY_RED_LIST_INFO=false -e BASE_DOMAIN=drt-test.homeoffice.gov.uk 340268328991.dkr.ecr.eu-west-2.amazonaws.com/drt/drt-v2:B${DRONE_BUILD_NUMBER} $${PLAY}
```

Comment:
we don't want to give the tests the production aws creds. I would suggest we use dummy values or add a default dummy value into the .conf files so we don't need to add them here